docs/spec/qualifiers.rst

```diff
@@ -313,7 +343,7 @@ Aliases & Concerns over verbosity
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Writing ``typing.Annotated`` everywhere can be quite verbose;
-fortunately, the ability to alias annotations means that in practice we
+fortunately, the ability to alias types means that in practice we
```

Comment:
I actually think this is a place where the previous wording was preferable; "annotation" here was not being used to refer to a metadata object.

Reply:
There is no ability to alias annotations, though, only to alias types. For example, this is illegal:

```
FinalInt = Final[int]
x: FinalInt = 1
```